src/server/config/database-config.js

```diff
@@ -0,0 +1,6 @@
+import Sequelize from 'sequelize';
+
+export default new Sequelize('passport', 'postgres', '1111', {
+  host: 'localhost',
+  dialect: 'postgres',
+});
```

Comment:
1. This is not a config. This file creates connection to database. Move it into utils folder and rename it into `CreateDBConnection.js`.
2. Use CamelCase instead kebab-case.
3. Use Config from `config/postgresConnection.json`, because we need single source of trust for database config.